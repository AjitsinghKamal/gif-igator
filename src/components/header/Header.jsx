import React from 'react';
import { Container } from 'style/Helpers';
import styled from 'styled-components';
import Logo from 'components/shared/logo/Logo';
import SearchBox from 'components/searchBox/SearchBox';
const Wrap = styled(Container)`
	position: fixed;
	left: 0;
	right: 0;
	top: 0;
	width: 100%;
	z-index: 10;
	display: flex;
	align-items: center;
	padding-top: 10px;
	padding-bottom: 10px;
	justify-content: space-between;
	background: ${props => props.theme.background};
`;

const Header = () => {
	return (
		<Wrap as="header">
			<Logo />
			<SearchBox />
		</Wrap>
	);
};

export default Header;
