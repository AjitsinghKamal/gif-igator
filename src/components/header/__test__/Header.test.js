import React from 'react';
import Header from '../Header';
import { shallow } from 'enzyme';
describe('Searchbox component', () => {
	it('renders correctly', () => {
		const wrapper = shallow(<Header />);
		expect(wrapper).toMatchSnapshot();
	});
});
