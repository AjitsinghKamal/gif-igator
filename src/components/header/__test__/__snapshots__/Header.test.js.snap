// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbox component renders correctly 1`] = `ShallowWrapper {}`;
